src/main/java/uk/gov/justice/hmiprobation/casesampler/services/RoAllocationAdjuster.kt

```diff
@@ -2,16 +2,11 @@ package uk.gov.justice.hmiprobation.casesampler.services
 
 import org.slf4j.LoggerFactory
 import uk.gov.justice.hmiprobation.casesampler.utils.Counter
-import uk.gov.justice.hmiprobation.casesampler.utils.Result
 import uk.gov.justice.hmiprobation.casesampler.utils.SampleSize
 import uk.gov.justice.hmiprobation.casesampler.utils.Type.DECREASED_FOR_RO
 import uk.gov.justice.hmiprobation.casesampler.utils.Type.INCREASED_FOR_RO
 
-data class RoSize(val ro: String, val size: SampleSize) {
-    constructor(pair: Pair<String, SampleSize>) : this(pair.first, pair.second)
-
-    fun toPair() = ro to size
-}
+typealias RoSize = Pair<String, SampleSize>
```

Comment:
Nice!

Reply:
the cost is that we lose the nice name of properties and just get `first` and `second`, which is why I reverted to destructuring in some of the method calls

`.sortedBy { it.ro }` -> `.sortedBy { (ro, _) -> ro }` rather than `.sortedBy { it.first }`